db/seeds.rb

```diff
@@ -37,5 +46,47 @@
   email: "jomo@example.com",
   password: "123456789", # high seructity!
   password_confirmation: "123456789",
-  role: Role.get(:superadmin)
-)
\ No newline at end of file
+  role: Role.get(:superadmin),
+  badge: Badge.get(:donor),
+  confirmed: true
+)
+User.create!(
```

Comment:
Just write a rake task to create (admin) users then instead of having to log in using accounts of other users, it only causes issues, especially when using this in production.